### ISSUE TITLE
Opening an OCaml project before having dependencies in .merlin file keeps giving errors even after adding the dependency

### ISSUE THREAD

**Issue by sharno** [reasonml-editor/vscode-reasonml#110]
How to reproduce:

- Open an OCaml project that has a file that imports any module (for example core `open Core`)
- The project should have no .merlin file or has the file but the .merlin file doesn't have `PKG core` at the time of opening it
- Add PKG core to the .merlin file
- The error keeps printing to the console with every edit to the document until restarting VSCode

The error messages: (the first one keeps repeating with edits - also they don't show what's wrong)
```
(node:65198) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: <unreachable>: error
(node:65198) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by jchavarri** [comment]
@sharno which OS are you on?

**Comment by sharno** [comment]
@jchavarri Mac Sierra

**Comment by emctoo** [comment]
Some problem, this happens when open a new *.ml/mli file, and f12 does not work

**Comment by freebroccolo** [comment]
> (node:65198) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: <unreachable>: error
> (node:65198) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

This part of the problem should be fixed in `1.0.6`. Getting the diagnostics properly refreshed when `.merlin` changes should be in an upcoming release.

**Comment by freebroccolo** [comment]
This should be fixed now that the merlin auto-refresh functionality is enabled.
